content/francineblanc/how-to-use-git-for-beginners.md

```diff
@@ -0,0 +1,219 @@
+2. Working on files in the Working Directory, then adding these to the Staging Area;
+3. Commmiting the changes from the Staging Area into the Repository.
+
+Let's unpack each of these steps.
+
+## Initialising Git
+To turn a Working Directory into a Git project, navigate to the directory in the terminal/command prompt and run the following command:
+```
+git init
+```
+This initialises a Git repository (which is just a name for a special folder called ```.git``` inside a project) on your local machine, and is required in order for Git to start tracking changes. This step might look something like this:
+
+![git-init](git-init.png)
+
+When ```git init``` is run inside the Git Tutorial folder, an empty Git repository is created inside of that directory. The repository is currently empty because Git doesn't know at this point which files it should track. Any untracked files can be viewed using the command:
+```
+git status
+```
+This will display untracked files in <span style="color:red">red</span>, and will show some useful output confirming that nothing has been added. For example, if ```git status``` is run in the Git Tutorial folder, which now contains a file called ```learning-git.txt``` the output looks like this:
+
+![git-status-untracked](git-status-untracked-files.png)
+
+You will see the output says ```On branch main``` which you can ignore at the moment. You'll also see a mysterious file called ```.DS_Store``` which is a file that is automatically created by Mac OS X. This file contains information about system configurations, so should not be committed as part of the Git workflow. To make sure this file is not added to the commit, a special file called ```.gitignore``` must be made in the Working Directory.
+
+### .gitignore
+The ```.gitignore``` file is a special file which generally lives in the root of the Working Directory. It contains a list of files and/or directories which a developer has added that Git should not track, and so will not be committed. These files may be log files, or particular modules in a project that shouldn't be tracked, or (for Mac users) files like ```.DS_Store```. Make sure to try and remember to add files which should not be tracked ***before*** they are committed.
+
+Now, with the ```.gitignore file``` made and the ```.DS_Store``` file added, running ```git status``` in the terminal shows the following output:
+
+![git-status-untracked-with-gitignore](git-status-untracked-with-ignore.png)
+
+## Making Changes and Adding Files to the Staging Area
+In order for Git to start tracking changes, files need to be added to the Staging Area. This can be done with the command:
+```
+git add <filename>
+```
+where filename is the name and extension of the file to be added to the staging area. Multiple files can also be added with the syntax:
+
+```
+git add <filename1> <filename2> <filename3>
+```
+
+If all files in the Working Directory need to be tracked, the syntax
+
+```
+git add .
+```
+can be used.
```

Comment:
```suggestion
If all files in the working directory need to be tracked, this syntax can be used:

```bash
git add .
```
```

This is another command we can use: `git add -A`. Also for `git add .`, this should be run at the root directory 👍